xml/deployment_hwrecommend.xml

```diff
@@ -23,6 +23,172 @@
   basis. If several processes are located on the same machine, the CPU, RAM,
   disk and network requirements need to be added up.
  </para>
+
+<sect1 xml:id="network-overview">
+  <title>Network Overview</title>
+  <para>
+    Ceph has several logical networks:
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+        A trusted internal network, the back-end network called the
+        the <literal>cluster network</literal>.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        A public client network called <literal>public network</literal>.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Client networks for gateways, these are optional.
+      </para>
+    </listitem>
+  </itemizedlist>
+   <para>
+     The trusted internal network is the back-end network between the OSD
+     nodes for replication and balancing, it needs roughly three times the
+     bandwidth as the public network.
```

Comment:
Hmm I need to nitpick this a little. I'd propose to change this to
```
The trusted internal network is the back-end network between the OSD nodes for replication,
re-balancing and recovery. Ideally this network has twice the bandwidth of the public network
since with default 3-way replication the primary OSD sends 2 copies to other OSDs via this network.

Reply:
I'd argue that.  From a simplicity perspective, it would be better to put all four ports into a single bond and continue to use VLANs.  This dramatically improves the serviceability by eliminating trying to figure out which physical port is some randomly assigned eth device.

Reply:
Fair enough regarding the VLANs, no objections againt that. However I'm arguing the cluster network does not need to be 3xbandwidth of the public network.

I think this is a change that should be added. I'm ok with recommending bonds and VLANs over physically separate networks.

Reply:
The text is saying 2xbandwith of public network, not 3 times.

Do you have a better rule of thumb? Or a suggestion for reformulation?

Reply:
Now I'm getting it - you propose to change "3x" to "2x", correct?

Reply:
correct. One replica wirth of data comes via the public network from the client to the primary. The primary then writes two replicas to other OSDs. All that under the assumption of 3way-replication of course.

Reply:
When I wrote 3 I was thinking of rebalancing as well. We can write 2 here - just wanted some rule of thumb.